penguin/system/runtime_event_ledger.py

```
@@ -0,0 +1,533 @@
+                )
+                """,
+                (batch,),
+            )
+            conn.commit()
+            current_size = self._database_size_bytes(conn)
+            if int(count) <= batch:
+                break
+        try:
+            conn.execute("PRAGMA incremental_vacuum")
+        except sqlite3.DatabaseError:
+            pass
```

Ruff (0.15.20)

[warning] Use `contextlib.suppress(sqlite3.DatabaseError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(sqlite3.DatabaseError): ...`

(SIM105)